Sources/Mantis/CropView/CropView.swift

```diff
@@ -247,15 +251,15 @@ class CropView: UIView {
         if rotationDial != nil {
             rotationDial?.removeFromSuperview()
         }
-        
-        var config = DialConfig.Config()
-        config.backgroundColor = .clear
-        config.angleShowLimitType = .limit(angle: CGAngle(degrees: 40))
-        config.rotationLimitType = .limit(angle: CGAngle(degrees: 45))
-        config.numberShowSpan = 1
+
+        var dailConfig = DialConfig.Config()
+        dailConfig.backgroundColor = .clear
+        dailConfig.angleShowLimitType = .limit(angle: CGAngle(degrees: 40))
+        dailConfig.rotationLimitType = config.rotationLimit
+        dailConfig.numberShowSpan = 1
```

Comment:
I think if we pass a DialConfig, we don't need to create a local one here.

Reply:
DailConfig and Config are not the same... `dailConfig` was original called config. It was always local. But because we want to have some settings from Mantis.Config I changed the name to prevent confusion. We could also do the whole DailConfig setup in Config and then pass it along.

Reply:
My orginal name `config` is not good, `dialConfig` is better. I think `dailConfig` is a typo
But since you want to cumtomize DialConfig, I think you can build DailConfig outside CropView with Config.rotationLimit, then pass it to CropView.